product_code/course_service/CHANGELOG.md

```diff
@@ -6,7 +6,11 @@
 
 # [v0.11.1](https://github.com/upb-uc4/University-Credits-4.0/compare/course-v0.11.0...course-v0.11.1) (2020-11-06)
```

Comment:
This is part of v0.12.1.

Reply:
Thank you for the pointer, it is now fixed